webhook/webhook.groovy

```diff
@@ -512,8 +512,8 @@ class WebHook {
                 for (def webhookListener : webhookListeners) {
                     try {
                         if (!webhookListener.isAsync())
-                            if (eventPassedFilters(event, json, webhook))
-                                callPost(webhookListener.url, getFormattedJSONString(json, webhook))
```

Comment:
bugfix:  getFormatterJSONString needs an event parameter but none is passed, should pass in event